frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -225,8 +232,32 @@ export const EditZing = () => {
       })
   }
 
+  const selectedGroups = studentGroups.filter((group) =>
+    selectedGroupNumbers.includes(group.groupNumber)
+  )
+  console.log(selectedGroups)
+
+  // Open and close the 'Send email to' menu drop down
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const menuOpen = Boolean(anchorEl)
+  const handleMenuOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+  const handleMenuClose = () => {
+    setAnchorEl(null)
+  }
+
+  const handleSelectNewlyMatched = () => {
+    setSelectedGroupNumbers(
+      studentGroups
+        .filter((group) => group.shareMatchEmailTimestamp === null)
+        .map((group) => group.groupNumber)
+    )
+    handleMenuClose()
+  }
```

Comment:
Is the definition of newly matched just "which one did not have an email get sent to"? I'm just wondering since I had no idea what the basis for the criterion is.

Reply:
My understanding is yes, that groups are considered newly matched if they haven't been sent an email telling them that they are matched with each other yet.